src/borg/repository.py

```diff
@@ -843,6 +850,8 @@ def report_error(msg):
             transaction_id = self.get_transaction_id()
             current_index = self.open_index(transaction_id)
             logger.debug('Read committed index of transaction %d', transaction_id)
+        except self.AtticRepository as exc:
+            raise
```

Comment:
What would be the point in re-raising? Since it's an `Error` the traceback is not normally shown, so there's little point in attaching a new traceback?